nntrainer/graph/network_graph.cpp

```diff
@@ -212,98 +214,90 @@ int NetworkGraph::realizeMultiOutputType(
   return status;
 }
 
-/**
- * @fixme: the implementation assumes loss layer should always be at the last
- * layer and the there is only one loss, this assumption is not true
- */
 int NetworkGraph::addLossLayer(const LossType loss_type) {
   int status = ML_ERROR_NONE;
-  auto const &last_node = LNODE(graph.getSortedNode(graph.size() - 1));
-  auto last_layer_node = getSortedLayerNode(graph.size() - 1);
 
-  if (last_layer_node->getObject()->requireLabel()) {
-    return status;
-  }
+  for (unsigned int i = 0; i < graph.getNumOutputNodes(); ++i) {
+    auto output_layer_node = LNODE(graph.getOutputNode(i));
 
-  if (loss_type == LossType::LOSS_NONE) {
-    return status;
-  }
+    if (output_layer_node->getObject()->requireLabel()) {
+      continue;
```

Comment:
so, if some loss layers are added as layers in the model, then this continue will lead to `loss` being added after all the output nodes which do not contain loss nodes. Is this intended?

Reply:
I think you want to say that in that case it makes an error and that's what we've discussed. In this pr I didn't handle such cases. Maybe in later pr I will take care of.

Reply:
cool.